docs/history.rst

```diff
@@ -0,0 +1,2 @@
+History
+========
```

Comment:
Should this be added? I think the commit history is usually good enough

Reply:
Not sure, wannted some place to track changes. But i think i want to use the "generate release notes" feature in GH insted.

Reply:
Removed it.